app/uk/gov/hmrc/gform/submission/destinations/DmsSubmitter.scala

```diff
@@ -86,26 +84,7 @@ class DmsSubmitter(
         success(
           Envelope(envelope.files.map(f => File(FileId(f.fileId), FileStatus.Available, f.fileName, f.length)))
         )
-      _ <- success(logFileSizeBreach(submission.envelopeId, envelopeDetails.files))
       _ <- formService.updateFormStatus(submissionInfo.formId, Submitted)
     } yield ()
   }
-
-  /** This log line is used in alert-config, to trigger a pager duty alert when files sizes exceeds the threshold
-    * value (currently 50 MB)
-    *
-    *  1 MB = 1000000 Bytes in base 10 system
-    * @param files
-    */
-  private def logFileSizeBreach(envelopeId: EnvelopeId, files: List[File]) = {
-    val totalFileSize = files.map(_.length).sum
-    val totalFileSizeMB = Math.ceil(totalFileSize / 1000000.toDouble)
-    val thresholdMB = 100
-    logger.info(
-      s"GForm DMS submission attachments size is $totalFileSize B (rounded to $totalFileSizeMB MB) [envelopeId=${envelopeId.value}]"
-    )
-    if (totalFileSizeMB > thresholdMB) {
-      logger.warn(s"GForm DMS submission attachments size exceeds $thresholdMB MB [envelopeId=${envelopeId.value}]")
```

Comment:
This is used [here](https://github.com/hmrc/alert-config/blob/main/src/main/scala/uk/gov/hmrc/alertconfig/configs/Gforms.scala#L21). 

Why are are we removing it?

Reply:
After object-store integration, I thought this check was unnecessary. I think there is filesize control in frontend. Also  threshold value is hardcoded. val thresholdMB = 100

Reply:
Reverted